docs/manage-statistics-in-gravitino.md

```diff
@@ -0,0 +1,285 @@
+---
+title: "Manage statistics in Gravitino"
+slug: /manage-statistics-in-gravitino
+date: 2025-08-21
+keyword: statistics management, statistics, statistic, Gravitino
+license: This software is licensed under the Apache License version 2.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+
+Starting from 1.0.0, Gravitino introduces statistics of tables and partitions.
+
+This document provides a brief introduction by using both Gravitino Java client and
+REST APIs. If you want to know more about the statistics system in Gravitino, please refer to the
+Javadoc and REST API documentation.
+
+Statistics only support the custom statistics, which names must start with `custom-`.
+Gravitino will support build-in statistics in the future.
+
+
+## Metadata object Statistic operations
+
+### Update statistics of metadata objects
+
+You can update the statistics of a metadata object by providing the statistics key and value.
+Now only table statistics can be updated.
+
+The request path for REST API is `/api/metalakes/{metalake}/objects/{metadataObjectType}/{metadataObjectName}/statistics`.
+
+<Tabs groupId='language' queryString>
+<TabItem value="shell" label="Shell">
+
+```shell
+curl -X PUT -H "Accept: application/vnd.gravitino.v1+json" \
```

Comment:
What's the behavior of updating an existing statistics key?  If the existing key belongs to Gravitino build-in table statistics, what's the behavior then?